PostBoard.Api/Program.cs

```diff
@@ -1,14 +1,19 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using PostBoard.Api;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-builder.Services.AddDbContext<PostBoardContext>(options =>
-    options.UseNpgsql("Host=localhost;Port=5432;Database=postdb;UserName=postgres;Password=rootroot"));
+builder.Configuration.AddJsonFile("appsettings.json");
+builder.Services.AddDbContext<PostBoardContext>((serviceProvider, options) =>
+{
+    var connectionString = builder.Configuration.GetConnectionString("PostBoardContext");
+    options.UseNpgsql(connectionString);
```

Comment:
Write it in one line:
`options.UseNpgsql(builder.Configuration.GetConnectionString("PostBoardContext");`